### DISCUSSION TITLE
Preliminary Exam code

### DISCUSSION THREAD

**Discussion by croberts808** [comp426-2023-spring/syllabus#4] [General]
I got an email that said "Some of the information at the end of some of your preliminary exams was malformed and you did not receive an exam code, so hopefully the information below will appear correctly."

I checked the github repository, and I had changed the exam code value correctly, so I am not sure what I did wrong. 

Anyone else get this email and know how to fix it?

Thanks!

**Comment by jdmar3** [comment]
So, this was an email that I automated Qualtrics to send, and of course it sent it without actually including the information. 😑  

It's nothing that you need to worry about. If you didn't get the exam code, we'll talk about that in class today. I'm going to use it as an example of how to write a good, actionable issue. So, nothing to worry about, and stay tuned for more.
